modules/github/__init__.py

```diff
@@ -521,8 +525,8 @@ def issue_comment(self, full_name, data):
         commenter = utils.irc.bold(data["comment"]["user"]["login"])
         url = self._short_url(data["comment"]["html_url"])
         return ["[%s #%d] %s %s on: %s - %s" %
-            (type, number, commenter, COMMENT_ACTIONS[action], issue_title,
-            url)]
+            (type, number, commenter, COMMENT_ACTIONS.get(action, "commented"),
```

Comment:
Wrong way to fix this. what was the `action` that wasn't being found in `COMMENT_ACTIONS`?

Reply:
commented

Reply:
put that in `COMMENT_ACTIONS` as `"commented": "commented"`

Reply:
There's a slight problem, the webhook sends created if you make a comment, not commented, so unless you want it to fail over?

Reply:
then add `"created": "commented"` to `COMMENT_ACTIONS`

Reply:
That's what it currently is

Reply:
so why do we have to `.get`?

Reply:
COMENT_ACTIONS[action] seemed to implode me, that might not be the case now, but it's the reason I changed it.